tests/Resources/Admin/SeoAwareContentAdmin.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SonataAdminBundle\Tests\Resources\Admin;
+
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\DoctrinePHPCRAdminBundle\Admin\Admin;
+
+class SeoAwareContentAdmin extends Admin
+{
+    protected $baseRouteName = 'cmf_seo_aware_content';
+    protected $baseRoutePattern = 'cmf/seo/seoawarecontent';
+
+    public function getExportFormats()
+    {
+        return array();
+    }
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->addIdentifier('id', 'text')
+            ->addIdentifier('title', 'text')
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('form.group_general')
+                ->add('name', 'text')
+                ->add('title', 'text')
+                ->add('body', 'textarea', array('required' => false))
```

Comment:
@dbu i just realized, that the point you are pointing to is a test admin, which doesn't have any purpose on SeoBundle. SeoBundle itself has its own form type `SeoMetadataType` which still is in the SeoBundle.